FusionMCPBridge/handlers/design/geometry_impl2.py

```
@@ -0,0 +1,642 @@
+            circularFeatInput = circularFeats.createInput(inputEntites, rootComp.zConstructionAxis)
+
+        circularFeatInput.quantity = adsk.core.ValueInput.createByReal(quantity)
+        circularFeatInput.totalAngle = adsk.core.ValueInput.createByString('360 deg')
+        circularFeatInput.isSymmetric = False
+        circularFeats.add(circularFeatInput)
+    except:
+        if ui:
+            ui.messageBox('Failed circular_pattern:\n{}'.format(traceback.format_exc()))
+
+
+def undo(design, ui):
```

Ruff (0.14.10)

Unused function argument: `design`

(ARG001)